pkg/adaptation/plugin.go

```diff
@@ -184,7 +220,7 @@ func (r *Adaptation) getPluginConfig(id, base string) (string, error) {
 
 // Check if the plugin is external (was not launched by us).
 func (p *plugin) isExternal() bool {
-	return p.cmd == nil
+	return p.cmd == nil || p.wasm != nil
```

Comment:
Do we want to treat WASM plugins like 'external' ones ? 'External' feels like a bit of a misleading misnomer, and maybe this should have been `(!)isPreLaunched()`. Anyway, semantically WASM plugins feel to me closer to `!isExternal()` plugins: they are discovered and automatically started (effectively) by the runtime during startup.

Reply:
I changed the logic to exclude them from "external"